spec/mailers/previews/user_mailer_preview.rb

```diff
@@ -0,0 +1,14 @@
+class UserMailerPreview < ActionMailer::Preview
+
+  def account_activation
+    user = User.first
+    user.activation_token = User.new_token
+    UserMailer.account_activation(user)
+  end
+
+  def password_reset
+    user = User.first
```

Comment:
Same issue here

Reply:
I changed them to a test user, what do you think?

Reply:
It looks better, GJ 👍

Reply:
Thank you!